using-ocean-market/remove-liquidity-using-etherscan.md

```diff
@@ -0,0 +1,44 @@
+# Remove liquidity using Etherscan
+
+## Get your balance of pool share tokens
+
+1. Go to the pool's Etherscan/Polygonscan page. You can find it by inspecting your transactions on your account's Etherscan page under _Erc20 Token Txns_.
+2. Click _View All_ and look for Ocean Pool Token (OPT) transfers. Those transactions always come from the pool contract, which you can click on.
+3. On the pool contract page, go to _Contract_ -> _Read Contract_.
+
+![](../.gitbook/assets/remove-liquidity-1.png)
+
+4\. Go to field `20. balanceOf` and insert your ETH address. This will retrieve your pool share token balance in wei.
+
+![](../.gitbook/assets/remove-liquidity-2.png)
+
+5\. Use this number as value for `poolAmountIn` when exiting the pool.
```

Comment:
Is more like: Copy this number as later you will use it as the poolAmountIn parameter. Right?

Reply:
yes. updated the step for more clarity.

> I suggest the following adjustments to the steps you added:
> 
> 1. Go to the pool's Etherscan/Polygonscan page. You can find it by inspecting your transactions on your account's Etherscan page under Erc20 Token Txns.
> 2. Click View All and look for Ocean Pool Token (OPT) transfers. Those transactions always come from the pool contract, which you can click on.
> 3. On the pool contract page, go to Contract -> Read Contract.
> 4. Go to field 20. `balanceOf` and insert your ETH address. This will retrieve your pool share token balance in wei.
> 5. Copy this number as you will use it as `poolAmountIn` when exiting the pool.
> 6. Go to field 56. `totalSupply` to get the total `amount` of pool shares, in (wei)[A link to what wei means maybe??].
> 7. Divide the number by 2 to get the maximum of pool shares you can send in one pool exit transaction. If your number retrieved in former step is bigger, you have to send multiple transactions.
> 8. Go to Contract -> Write Contract and connect your wallet. Be sure to have your wallet connected to network of the pool.
> 9. Go to the field 5. exitswapPoolAmountIn...
> 10. Confirm transaction in Metamask
> 
> You just removed liquidity from the pool. Check your wallet. If you weren't able to remove your entire liquidity, repeat the steps from 4 to 10. @jamiehewitt15 @akshay-ap @kremalicious what do you think? Personally I think it's easier to follow the steps in this way

For me, the existing content split into 3 main sections is clear:
- Get your balance of pool share tokens
- Get maximum to remove
- Exit pool
Because, going from 1 to 3 involves switching between `read` mode to `write` mode in etherscan's interface.

But, I am fine with making steps numbered from 1 to 10.